test/python/gemm/frost/test_moe_grouped_block_scale_matmul_fwd_swiglu.py

```
@@ -184,6 +184,39 @@ def test_analyzer_detects_dual_moe_grouped_block_scale_matmul_fwd() -> None:
+            g.moe_grouped_matmul(
+                g.block_scale_dequantize(input=tok, descale=sfa[i], block_size=[1, bs]),
+                g.block_scale_dequantize(input=w, descale=sfb, block_size=[bs, 1]),
+                fto,
+                mode=cudnn.moe_grouped_matmul_mode.NONE,
+                compute_data_type=cudnn.data_type.FLOAT,
+                name=f"moe{i}",
+            )
+        )
+    Y = g.mul(a=g.swish(input=outs[0]), b=outs[1])
+    Y.set_output(True).set_data_type(cudnn.data_type.BFLOAT16)
+    with pytest.raises(ValueError, match="A operand .* dequantized more than once"):
```

Ruff (0.16.2)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)